build-tools/scripts/TestApks.targets

```diff
@@ -347,6 +347,15 @@
         LabelSuffix="-$(Configuration)$(TestsFlavor)"
         ContinueOnError="ErrorAndContinue"
     />
+    <PropertyGroup>
+      <ApkDiffPath>$(XAInstallPrefix)xbuild\Xamarin\Android\$(HostOS)\apkdiff</ApkDiffPath>
+      <ApkDiffPath Condition=" !Exists('$(ApkDiffPath)') ">$(MonoAndroidBinDirectory)\apkdiff</ApkDiffPath>
+    </PropertyGroup>
+    <Exec
+        Condition=" Exists('$(ApkDiffPath)') "
+        Command="&quot;$(ApkDiffPath)&quot; -v -c &quot;HEAD/`git branch --show-current`: `git rev-parse HEAD`&quot; -s &quot;%(_AllArchives.Identity)&quot;;mv &quot;%(_AllArchives.RelativeDir)\%(_AllArchives.Filename).apkdesc&quot; &quot;%(_AllArchives.RelativeDir)\%(_AllArchives.Filename)-$(Configuration)-$(TestsFlavor).apkdesc&quot;"
```

Comment:
I *think* that `-$(TestsFlavor)` can instead be `$(TestsFlavor)`.

Right now the [`xa-test-results-10.2.99.215_739a18a-Darwin-Release.zip`](https://app.vssps.visualstudio.com/_signedin?realm=artprodscussu1.artifacts.visualstudio.com&protocol=wsfederation&reply_to=https%3A%2F%2Fartprodscussu1.artifacts.visualstudio.com%2FA011b8bdf-6d56-4f87-be0d-0092136884d9%2F0bdbc590-a062-4c3f-b0f6-9383f67865ee%2F_apis%2Fartifact%2FcGlwZWxpbmVhcnRpZmFjdDovL2RldmRpdi9wcm9qZWN0SWQvMGJkYmM1OTAtYTA2Mi00YzNmLWIwZjYtOTM4M2Y2Nzg2NWVlL2J1aWxkSWQvMzQ0OTMxNy9hcnRpZmFjdE5hbWUvVGVzdCtSZXN1bHRzKy0rQVBLK0luc3RydW1lbnRhdGlvbistK21hY09T0%2Fcontent%3Fformat%3Dfile%26subPath%3D%252Fxa-test-results-10.2.99.215_739a18a-Darwin-Release.zip&hid=011b8bdf-6d56-4f87-be0d-0092136884d9) file contains filenames such as:

```
% find . -iname \*.apkd\*
./Xamarin.Android.EmbeddedDSO_Test-Signed-Release-.apkdesc
./Mono.Android_Tests-Signed-Release--Profiled-Aot.apkdesc
./Xamarin.Forms_Performance_Integration-Signed-Release-.apkdesc
./Xamarin.Forms_Performance_Integration-Signed-Release--Bundle.apkdesc
./Mono.Android_Tests-Signed-Release-.apkdesc
./Mono.Android_Tests-Signed-Release--Aot.apkdesc
./Xamarin.Forms_Performance_Integration-Signed-Release--Aot.apkdesc
./Xamarin.Android.Locale_Tests-Signed-Release-.apkdesc
./Mono.Android_Tests-Signed-Release--Bundle.apkdesc
./Xamarin.Android.JcwGen_Tests-Signed-Release-.apkdesc
./Xamarin.Android.Locale_Tests-Signed-Release--Aot.apkdesc
./Xamarin.Forms_Performance_Integration-Signed-Release--Profiled-Aot.apkdesc
./Mono.Android_TestsMultiDex-Signed-Release-.apkdesc
./Xamarin.Android.Locale_Tests-Signed-Release--Profiled-Aot.apkdesc
```

Note that many filenames have a trailing `-`, e.g. `Xamarin.Android.JcwGen_Tests-Signed-Release-.apkdesc`, or have `--`, e.g. `Xamarin.Android.Locale_Tests-Signed-Release--Aot.apkdesc`.  I believe using *just* `$(TestsFlavor)` would fix this.